### PR TITLE
import setting Ui creation

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Import setting UI
Screenshot 2023-11-07 at 6 25 12 PM
Screenshot 2023-11-07 at 6 25 21 PM

File: src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts [hunk 1]
```diff
@@ -85,6 +85,12 @@ export class Sage300ImportSettingsComponent implements OnInit {
 
   showDependentFieldWarning: boolean;
 
+  redirectLink: string = Sage300Link.IMPORT_SETTING;
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts:88]: name it supportArticleLink

File: src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts [hunk 2]
```diff
@@ -271,6 +277,10 @@ export class Sage300ImportSettingsComponent implements OnInit {
     this.isPreviewDialogVisible = visible;
   }
 
+  closeDialog() {
+    this.isPreviewDialogVisible = false;
+  }
+
   showOrHideAddButton() {
```
Comment (ashwin1111) [inline src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts:284]: not required in this file, this is there in shared component already

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts (hunk 2 above)
Comment (DhaaraniCIT) [reply src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts:284]: we dont have it in shared component

Comment (ashwin1111) [reply src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts:292]: Screenshot 2023-11-07 at 7 40 18 PM

Comment (DhaaraniCIT) [reply src/app/integrations/sage300/sage300-shared/sage300-import-settings/sage300-import-settings.component.ts]: you were telling `showOrHideButton`,  i thought you mention `closeDialog` funtion

**Pull request merged**